Tests/TLSEvaluationTests.swift

```diff
@@ -143,11 +142,11 @@ class TLSEvaluationExpiredLeafCertificateTestCase: BaseTestCase {
 
         // Then
         XCTAssertNotNil(error, "error should not be nil")
-
-        if let error = error?.asAFError {
-            XCTAssertTrue(error.isServerTrustEvaluationError, "should be .certificatePinningFailed")
+        XCTAssertEqual(error?.isServerTrustEvaluationError, true)
```

Comment:
Newline? Same comment throughout this file.

Reply:
Added.